Dev

desht opened the pull request:
Some changes necessary for upcoming FTB Quests & FTB XMod Compat 1.20 releases

mezz commented on src/main/java/com/feed_the_beast/ftblib/FTBLib.java:
```diff
@@ -31,7 +31,7 @@
 		version = FTBLib.VERSION,
 		acceptableRemoteVersions = "*",
-		dependencies = "required-after:forge@[0.0.0.forge,)"
+		dependencies = "required-after:forge@[0.0.0.forge,);after:jei@[4.6.0,);"
```
Note this minimum JEI version is from over a year ago, so it should not cause any problems.
It is just here in case someone has a very ancient JEI, then the plugin would crash.

LatvianModder commented:
Thanks! I will probably just remove that scaling thing, I don't think anyone actually uses it

desht committed:
chore: move fluid texture & color querying methods, make public

Out of SelectFluidScreen, into ClientUtils. Useful for FTB Quests fluid
icon rendering.

desht committed:
fix: subgroups should not inherit the parent's save-callback

Otherwise, it gets run multiple times, since all groups have their
save() method called.

desht committed:
chore: add default sync() method to StageProvider
